samples/boards/stm32/i2c_timing/src/main.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2023 STMicroelectronics
+ *
```

Comment:
Copyright (c) 2024 STMicroelectronics

Reply:
this PR started last year, so 2023 was pertinent at that time,
should we really  update this comment since this is a "internal" sample